vuepress/docs/next/docs/concepts/pda-architecture.md

```diff
@@ -41,28 +34,26 @@ The source code for this component:
 
 ## Sidecar
 
-The sidecar is another application (docker container) deployed in the same Pod
-as the PDA API. It exposes services to manage connection details, which will be
-stored as secrets in Kubernetes. The PDA API communicates with the sidecar to
+The sidecar is another application (Docker container) deployed in the same Pod
```

Comment:
I'm not sure how to track it but I think the sidecar has now been removed in 6.3.2. @Lyd1aCla1r3, once you get your local copy working, let's look at it together. If it's actually gone, we can remove all reference to the sidecar in the /next docs.